custom_components/husqvarna_automower/entity.py

```diff
@@ -61,6 +61,16 @@ def device_info(self) -> DeviceInfo:
             suggested_area="Garden",
         )
 
+    @property
```

Comment:
Is it possible to move this property to `camera.py`, because it's used only there and is not needed for all the other entities?

Reply:
I'd prefer to leave it there, I have other sensors that use that property and moving it to the camera makes any future PRs even more work.